api/resolvers/user.js

```diff
@@ -424,7 +435,8 @@ export default {
             confirmedAt: {
               gt: lastChecked
             },
-            isHeld: null
+            isHeld: null,
+            actionType: null
```

Comment:
In the future, `actionType` will be `BUY_CREDITS`?

Reply:
Yes, good point. All invoices should have an `actionType` and run through the state machine in the hopefully near future.